src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant-configurator/variant-configurator.twig

```diff
@@ -25,6 +25,12 @@
         {% set selectedValue = data.selectedAttributes[name] | default %}
         {% set selectedValue = selectedValue in values ? selectedValue : null %}
         {% set isAvailable = true %}
+        {% set optionResetUrl = '' %}
+
+        {% for selectedAttributeName, selectedAttributeValue in data.selectedAttributes | filter((value, key) => key != name) %}
+            {% set selectedUrlParameter = [('attribute[' ~ selectedAttributeName ~ ']')| url_encode, selectedAttributeValue] | join('=') %}
```

Comment:
Why do we need an array here instead of a string?

Reply:
We don't, string can be used